wardrobe/kitties/src/lib.rs

```diff
@@ -1,18 +1,36 @@
-//! An NFT game inspired by cryptokitties.
-//! This is a game which allows for kitties to be bred based on a few factors
-//! 1.) Mom and Tired have to be in a state where they are ready to breed
-//! 2.) Each Mom and Dad have some DNA and the child will have unique DNA combined from the both of them
-//!     Linkable back to the Mom and Dad
-//! 3.) The game also allows Kitties to have a cooling off period inbetween breeding before they can be bred again.
-//! 4.) A rest operation allows for a Mom Kitty and a Dad Kitty to be cooled off
+//! An NFT game inspired by Cryptokitties.
+//! This is a game that allows for the creation, breeding, and updating of the name of kitties.
 //!
-//! In order to submit a valid transaction you must strutucture it as follows:
-//! 1.) Input must contain 1 mom and 1 dad
-//! 2.) Output must contain Mom, Dad, and newly created Child
-//! 3.) A child's DNA is calculated by:
+//! ## Features
+//!
+//! - **Create:** Generate a new kitty.
+//!   To submit a valid transaction for creating a kitty, adhere to the following structure:
+//!   1. The input must be empty.
+//!   2. The output must contain only the newly created kitties as a child.
+//!
+//!    **Note 1:** Multiple kitties can be created at the same time in the same transaction.
+//!
+//! - **Update Name:** Modify the name of a kitty.
+//!   To submit a valid transaction for updating a kitty's name, adhere to the following structure:
+//!   1. The input must be the kitty to be updated.
+//!   2. The output must contain the kitty with the updated name.
+//!
+//!    **Note 1:** All other properties, such as DNA, parents, free breedings, etc., must remain unaltered in the output.
+//!    **Note 2:** The input and output kitties must follow the same order.
+//!
+//! - **Breed:** Breed a new kitty using mom and dad based on the factors below:
+//!   1. Mom and Dad have to be in a state where they are ready to breed.
+//!   2. Each Mom and Dad have some DNA, and the child will have unique DNA combined from both of them, linkable back to the Mom and Dad.
```

Comment:
```suggestion
//!   2. The child's unique DNA combined from Mom's and Dad's, linkable back to them.
```
I think it is redundant to state that Mom and Dad have some DNA, because it is mandatory